client/src/App.js

```diff
@@ -6,6 +6,7 @@ import { theme } from './themes/theme';
 // import LandingPage from './pages/Landing';
 import Signup from './pages/auth/Sign-up';
 import Login from './pages/auth/Login';
+// import UserInfo from './components/Profile/UserInfo';
```

Comment:
You can delete the commented code: if you need it back later on, you can just add it back in!